client/payment-details/payment-method/index.js

```diff
@@ -20,6 +20,8 @@ import IdealDetails from './ideal';
 import P24Details from './p24';
 import SepaDetails from './sepa';
 import SofortDetails from './sofort';
+import AffirmDetails from './affirm';
+import AfterpayClearpayDetails from './afterpay-clearpay';
 
```

Comment:
If we sort imports here we will avoid conflicts with #7282

Reply:
Done here https://github.com/Automattic/woocommerce-payments/pull/7345/commits/07c70fac9f49ea458d328ddb925816251f2f49a9 thanks!